docs/directory-structure.md

```diff
@@ -1,241 +1,206 @@
+    point right now, but it’s something to keep in mind as a long-term
+    goal.
+
+[639-3]: https://iso639-3.sil.org/code_tables/639/data
+
+### Proposed layout
+
+    $repo
+    ├── .git/
+    ├── package.json                # dependencies for bundlers, JS/CSS frameworks
+    ├── Pipfile
+    ├── arpeng-manage.py            # manage.py scripts are at the top-level for easy access
+    ├── crkeng-manage.py
+    ├── crkfra-manage.py
+    ├── cwdeng-manage.py
+    ├── srseng-manage.py
+    └── src/
+        ├── CreeDictionary/         # existing code, eventually goes away
+        │   ├── __init__.py
+        │   ├── API/                # this name goes away :(
+        │   └── CreeDictionary/     # this goes away too
+        │       ├── __init__.py
+        │       ├── models.py
+        │       ├── views.py
+        │       └── …
+        │
+        ├── morphodict/             # python package for language-independent code
+        │   ├── __init__.py
+        │   ├── cvd/
+        │   ├── lexicon/            # django app with primary database tables
+        │   │   ├── __init__.py
+        │   │   ├── models.py
+        │   │   ├── management/
+        │   │   │   └── commands/
+        │   │   │       └── dictimport.py
+        │   │   ├── parser.py
+        │   │   ├── test_parser.py
+        │   │   └── testdata/       # Use `testdata` directories for test data
+        │   ├── paradigm_filler/
+        │   ├── frontend/
+        │   │   ├── foo.js
+        │   │   ├── foo.css
+        │   ⋮   └── …
+        │
+        ├── crkeng/                 # python package for itwêwina
+        │   ├── __init__.py
+        │   ├── cypress/
+        │   │   └── …
+        │   ├── docker/
+        │   │   └── …
+        │   ├── res/                # Resources go here
+        │   │   ├── altlabels.tsv
+        │   │   ├── dictionaries/
+        │   │   ├── fst/
+        │   │   └── layouts/
+        │   ├── site/               # Django project
+        │   │   ├── __init__.py
+        │   │   ├── settings.py
+        │   │   └── urls.py
+        │   └── frontend/           # Not a python package
+        │       ├── foo.js
+        │       └── foo.css
+        │
+        ├── cwdeng/                 # python package for Woods Cree dictionary
+        │   ├── __init__.py
+        │   ├── cypress/
+        │   │   └── …
+        │   ├── docker/
+        │   │   └── …
+        │   ├── res/
+        │   │   ├── altlabels.tsv
+        │   │   ├── dictionaries/
+        │   │   ├── fst/
+        │   │   └── layouts/
+        │   ├── site/               # Django project
+        │   │   ├── __init__.py
+        │   │   ├── settings.py
+        │   │   └── urls.py
+        │   └── frontend/           # Not a python package
+        │       ├── foo.js
+        │       └── foo.css
+        │
+        ├── arpeng/
+        ├── crkfra/
+        ├── cr_shared               # for code and resources shared between Cree dialects
+        └── srseng/
```

Comment:
Is there a plan for where tests should live?
Does each individual service have its own tests as well as a larger set of tests for all the services?

Reply:
I see the explanation below now. I'm still wondering if each project should have a dedicated `tests` folder?

Reply:
I added some more comments about testing. We were hoping to not have any separate `tests` folders, but to have the tests mixed in, but if you really like having `tests` folders we can definitely talk about that.

Some of the testing stuff is pretty speculative at this point because it looks like it could turn out to be a bit of a pain to wrangle pytest to run tests across multiple django projects at once. If there turns out to be some way to organize the tests that makes running them much easier, that could end up overriding any other preferences.

Reply:
That should be fine. I think I see the thought process behind deciding where the tests go.